src/arduino/blinkende_lysdiode/blinkende_lysdiode.md

```diff
@@ -112,34 +113,46 @@ Det første programmet skal blinke med lysdioden.
 + Skriv denne koden:
 
   ```cpp
-  // Dette er porten vi har koblet lysdioden til
-  int led = 13;
+  int led = 8;
 
   void setup(){
-    // Porten til lysdioden (13) skal være utgang
     pinMode(led, OUTPUT);
   }
 
   void loop(){
-    // Skru på lysdioden
     digitalWrite(led, HIGH);
-    // Vent 1000 millisekund
     delay(1000);
-    // Skru av lysdioden
     digitalWrite(led, LOW);
-    // Vent 1000 millisekund
     delay(1000);
   }
   ```
 
-+ Trykk på play-knappen for å laste opp koden.
++ Trykk på "upload"-knappen for å laste opp koden. Denne sjekker først om koden
+er riktig, og så vil programmet ditt kjøre på arduinoen.
 + Blinker lysdioden?
 
+### Virker det ikke? {.protip}
+
+Hvis det ikke virker, så kan det hende at Arduino-programmet står på feil __port__
+og/eller __brett__. Da kan du sjekke disse to tingene:
+
++ Brett er satt riktig: __Tools -> Board -> Arduino/Genuino Uno__
+
++ Port er satt riktig:
+	+ Windows: __Tools -> Port -> COM1__ (kan være et annet tall)
+	![](port.png)
+	+ Mac: __Tools -> Port ->/dev/tty.usbmodem262471__ (kan være et annet tall)
++ LEDen er koblet riktig vei
+	+ Den korteste "foten" skal gå til GND
+
+Hvis dette ikke fungerer, kan du prøve å lukke programmet og åpne det igjen.
```

Comment:
Når jeg koblet til arduinoen for første gang, rett etter å ha startet Arduino-IDE for første gang, sto ikke den eneste porten jeg har valgt. Det sto i statusfeltet at COM1 var valgt, og COM1 fantes ikke på listen over porter. =)

På den andre datamaskinen jeg testet hadde jeg programmert en annen modell tidligere, og feilmeldingsskogen som kommer tilbake er ikke enkel å skjønne :p